internal/pkg/manifest/testdata/scheduled-job-no-timeout-or-retries.yml

```diff
@@ -0,0 +1,37 @@
+# The manifest for the "cuteness-aggregator" job.
+# Read the full specification for the "Scheduled Job" type at:
+#  https://github.com/aws/copilot-cli/wiki/Manifests#scheduled-job
+
+# Your job name will be used in naming your resources like log groups, ECS Tasks, etc.
+name: cuteness-aggregator
+# The "architecture" of the job you're running.
+type: Scheduled Job
+
+image:
+  # Docker build arguments. You can specify additional overrides here. Supported: dockerfile, context, args
+  build: ./cuteness-aggregator/Dockerfile
+
+# Number of CPU units for the task.
+cpu: 256
+# Amount of memory in MiB used by the task.
+memory: 512
+
+# The trigger for your job. You can specify a cron schedule or keyword (@weekly) or a rate (2h, 1h30m, 15m)
+schedule: "@weekly"
+# Optional. The number of times to retry the job before failing.
+#retries: 3
+# Optional. The timeout after which to stop the job if it's still running. You can use the units (h, m, s).
+#timeout: 1h30m
```

Comment:
should we have an empty line between `schedule` and `retries` and `timeout` for consistency? Also i feel like maybe we want to put `retries` and `timeout` under "# Optional fields for more advanced use-cases" since they are optional ones and not configured to be any default value if users don't set them. What do you think?

Reply:
I grouped these this way so they're logically connected. The other option might be to stick them in a `job` struct like:
```yaml
job:
  schedule: @weekly        # The trigger for your job. You can specify a cron schedule or keyword (@weekly) or a rate (2h, 1h30m, 15m)
#  retries: 3              # Optional. The number of times to retry the job before failing. 
#  timeout: 1h.            # Optional. The timeout after which to stop the job if it's still running. Must be a duration string (1h30m, 90s, etc)
```

What do you think?

Reply:
Maybe just leave it as it is now. Since the whole manifest is new for job, it doesn't make sense to me to have another section called `job`.

Reply:
btw I think we still should fix the spacing.